packages/ember-data/tests/integration/store_test.js

```diff
@@ -171,7 +171,6 @@ test("destroying the store correctly cleans everything up", function() {
   Ember.run(store, 'destroy');
 
   equal(car.get('person'), null, "expected car.person to no longer be present");
-  equal(person.get('cars'), undefined, "expected person.cars to be empty");
```

Comment:
Can you verify this? I tried it on jsbin and I couldn't find the change in behavior between release/canary.